src/sentry/static/sentry/app/components/sidebar/sidebarDropdown/index.jsx

```diff
@@ -179,7 +178,7 @@ const SentryLink = styled(Link)`
 `;
 
 const UserSummary = styled(Link)`
-  display: flex;
+  ${getMenuItemStyles}
```

Comment:
Where does this get imported/defined?

Reply:
`sidebarMenuItem` exports it. Wrapping it in that doesn't quite work since the menu item itself has the tabindex.